apps/tup-ui/src/auth/ProfileComponent.tsx

```diff
@@ -16,6 +17,7 @@ const ProfileComponent: React.FC = () => {
           <li>{profile.institution}</li>
           <li>{profile.country}</li>
         </ul>
+        <LogoutComponent />
```

Comment:
I can clean this up later, it was a hack during development. I'll probably place the LogoutComponent somewhere on the main app page rather than inside the ProfileCOmponent.

Reply:
For sure, I assume we'll find a good place for `LogoutComponent` once we start implementing the actual layouts.